uvtools/dspec.py

```diff
@@ -890,7 +893,10 @@ def gen_window(window, N, alpha=0.5, edgecut_low=0, edgecut_hi=0, **kwargs):
             w[edgecut_low:edgecut_hi] = getattr(windows, window)(N - Ncut)
         except AttributeError:
             raise ValueError("Didn't recognize window {}".format(window))
-
+    if normalization == 'rms':
+        w = w / np.sqrt(np.mean(np.abs(w)**2.))
```

Comment:
As a personal style preference, I'd write this as `w /= np.sqrt(np.mean(np.abs(w)**2.))` (and similarly below), but this is fine too.

Reply:
made this change.